applications/tests/test_applications_new_schema_mutations.py

```diff
@@ -36,9 +37,7 @@ def test_update_berth_application(
 
     assert berth_application.customer is None
 
-    executed = superuser_api_client.execute(
-        UPDATE_BERTH_APPLICATION_MUTATION, input=variables,
-    )
+    executed = api_client.execute(UPDATE_BERTH_APPLICATION_MUTATION, input=variables,)
```

Comment:
too bad black does not delete the trailing commas, after it combines several lines into one. 😞 could you delete the trailing commans manually? :derp: I mean the one after `input=variables`